### PR TITLE
fix: php 8.2 fixes

### PR DESCRIPTION
**Pull request by bradymiller**
This is a quick walk through of all the scripts I was able to run through.

Note that the more complicated/timeconsuming places in the codebase were not completed and have been reported as separate issues.

### CODE REVIEW

**Comment by bradymiller** [comment]
testing well and bringing in

File: interface/forms/fee_sheet/new.php [hunk 1]
```diff
@@ -1291,7 +1291,7 @@ function defaultPriceLevelChanged(sel) {
                                                 trim($bline['ndcqty']);
                                             }
                                             $justify    = $bline['justify'] ?? null;
-                                            $notecodes  = trim($bline['notecodes'] ?? null);
+                                            $notecodes  = trim($bline['notecodes'] ?? '');
```
Comment (stephenwaite) [inline interface/forms/fee_sheet/new.php:1294]: CzLY2Xbq

**Comment by juggernautsei** [comment]
> More complicated/time-consuming places in the codebase were not completed
Hi @bradymiller 
what are the more complicated places you are referring to?

**Comment by bradymiller** [comment]
Looks like they all have been addressed : https://github.com/openemr/openemr/issues?q=is%3Aissue+is%3Aclosed+8.2

**Pull request merged**